test/burials/00-all-fields.e2e.spec.js

```diff
@@ -0,0 +1,125 @@
+
+    // Previous Names page
+    client.waitForElementVisible('label[for$="serveUnderOtherNamesYes"]', Timeouts.normal);
+    PageHelpers.completePreviousNames(client, testData.data);
+    client.axeCheck('.main')
+      .click('.form-panel .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/military-history/previous-names');
+
+    // Benefit Selection page
+    client.waitForElementVisible('label[for$="claimedBenefits"]', Timeouts.normal);
+    client.assert.cssClassPresent('.progress-bar-segmented div.progress-segment:nth-child(4)', 'progress-segment-complete');
+    PageHelpers.completeBenefitSelection(client, testData.data);
+    client.axeCheck('.main')
+      .click('.form-panel .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/benefits/selection');
+
+    // Burial Allowance page -- conditional
+    if (testData.data['view:claimedBenefits'].burialAllowance) {
+      client.waitForElementVisible('label[for="root_burialAllowanceRequested"]', Timeouts.normal);
+      PageHelpers.completeBurialAllowance(client, testData.data);
+      client.axeCheck('.main')
+        .click('.form-panel .usa-button-primary');
+      E2eHelpers.expectNavigateAwayFrom(client, '/benefits/burial-allowance');
+    }
+
+    // Plot Allowance page -- conditional
+    if (testData.data['view:claimedBenefits'].plotAllowance) {
+      client.waitForElementVisible('input[name="root_placeOfRemains"]', Timeouts.normal);
+      PageHelpers.completePlotAllowance(client, testData.data);
+      client.axeCheck('.main')
+        .click('.form-panel .usa-button-primary');
+      E2eHelpers.expectNavigateAwayFrom(client, '/benefits/plot-allowance');
+    }
+
+
+    // Additional Information page
+    client.waitForElementVisible('select[name="root_claimantAddress_country"]', Timeouts.normal);
+    client.assert.cssClassPresent('.progress-bar-segmented div.progress-segment:nth-child(5)', 'progress-segment-complete');
+    PageHelpers.completeClaimantContactInformation(client, testData.data);
+    client.axeCheck('.main')
+      .click('.form-panel .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/claimant-contact-information');
+
+    // TODO: Test file upload
+
+    client.assert.cssClassPresent('.progress-bar-segmented div.progress-segment:nth-child(6)', 'progress-segment-complete');
+
+    // Review and Submit Page.
+    client
+      .waitForElementVisible('label[name="privacyAgreement-label"]', Timeouts.slow)
+      .pause(1000)
+      .click('input[type="checkbox"]')
+      .axeCheck('.main')
+      .click('.form-progress-buttons .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/review-and-submit');
+    client.expect.element('.js-test-location').attribute('data-location')
+      .to.not.contain('/review-and-submit').before(Timeouts.slow);
+
+    // Submit message
+    client.expect.element('.edu-benefits-submit-success').to.be.visible;
```

Comment:
`.edu-benefits-submit-success` seems weird here, but that's what it is, so... :man_shrugging: